vespajlib/src/main/java/com/yahoo/data/access/slime/SlimeAdapter.java

```diff
@@ -126,31 +131,27 @@ public String asString(String defaultValue) {
         return inspector.asData();
     }
     public void traverse(final com.yahoo.data.access.ArrayTraverser at) {
-        inspector.traverse(new com.yahoo.slime.ArrayTraverser() {
-                public void entry(int idx, com.yahoo.slime.Inspector inspector) { at.entry(idx, new SlimeAdapter(inspector)); }
-            });
+        inspector.traverse((ArrayTraverser) (idx, inspector) -> at.entry(idx, new SlimeAdapter(inspector)));
     }
     public void traverse(final com.yahoo.data.access.ObjectTraverser ot) {
-        inspector.traverse(new com.yahoo.slime.ObjectTraverser() {
-                public void field(String name, com.yahoo.slime.Inspector inspector) { ot.field(name, new SlimeAdapter(inspector)); }
-            });
+        inspector.traverse((ObjectTraverser) (name, inspector) -> ot.field(name, new SlimeAdapter(inspector)));
     }
     public com.yahoo.data.access.Inspector entry(int idx) { return new SlimeAdapter(inspector.entry(idx)); }
     public com.yahoo.data.access.Inspector field(String name) { return new SlimeAdapter(inspector.field(name)); }
     public Iterable<com.yahoo.data.access.Inspector> entries() {
-        final List<com.yahoo.data.access.Inspector> list = new ArrayList<>();
```

Comment:
Unrelated and also seems to slightly change behavior? Please remove this and put it in a separate PR to a module owner.

Reply:
Whoo, where did that come from!? I'll remove it.